### ISSUE TITLE
NTR: Spontaneous cerebrospinal fluid leak

### ISSUE THREAD

**Issue by pbuczkowicz** [obophenotype/human-phenotype-ontology#5489]
**Preferred term label:**
Spontaneous cerebrospinal fluid leak

**Synonyms**
CSF leak
SCSFL

**Definition (free text, please give PubMed ID)**
A spontaneous cerebrospinal fluid leak (SCSFL) is a spontaneous and unexplained leak of the cerebrospinal fluid from the dura surrounding either the brain (cranial leak) or spine (spinal leak). 

PMID: 19225307
PMID: 22929030

**Parent term (use hpo.jax.org/app)**
[HP:0002921] Abnormality of the cerebrospinal fluid

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM# 1300010
OMIM# 154700 
OMIM# 147900

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-6469-1475

**Comment by drseb** [comment]
Not sure if the parent term is the optimal choice. Isn't it rather an abnormality of the structure that usually holds the CSF within its borders? I think according to UBERON this is the "ventricular system of central nervous system" (UBERON:0005281)
@pnrobinson please advice

**Comment by pnrobinson** [comment]
I think we would prefer not to have the qualifier 'spontaneous'--this is a characteristic of the disease not of the phenotype. 
I agree with @drseb that perhaps this (https://hpo.jax.org/app/browse/term/HP:0002118) is a better parent term

**Comment by pbuczkowicz** [comment]
I would still suggest the parent term be [HP:0002921] Abnormality of cerebrospinal fluid, because one of its daughter terms, HP:0030998 Cerebrospinal fluid rhinorrhoea, should be categorized as a sub-type of CSF leak. Furthermore, the CSF leak can be spinal and therefore unrelated to any abnormality of the cerebral ventricles.

**Comment by pnrobinson** [comment]
@pbuczkowicz  This is true. But and we do need to remember that CSF rhinorrhea is one kind of leak. But the CSF itself is normal, even though it is being leaked, and so this is what Seb is referring to. We may need to rearrange a few things. Currently, we have Abn or CSF but we are moving towards having Abnormal morphology/Abn physiology terms. This is a case of Abnormal CSF physiology (also CSF pressure terms), and terms that have e.g increased CSF protein are morphology. I need to run now but we will think about this and present a summary here shortly.

**Comment by pbuczkowicz** [comment]
@pnrobinson If there is a rearrangement for the relationships between these terms to make sense under a different parent term then I am in agreement. 

Also, if spontaneous is being dropped from the preferred term label, can it still be listed as a synonym? There are many examples in the literature of the entire term "Spontaneous cerebrospinal fluid leak" or "SCSFL" being used.

Thank you.

**Comment by pnrobinson** [comment]
Leak is physiology in a sense, but essentially there is always a defect in the meninges, so I think for now we can put this term under here:
https://hpo.jax.org/app/browse/term/HP:0010651

**Comment by pnrobinson** [comment]
Adding term as HP_0032934
